content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/resupply_warehouses.rst

```diff
@@ -2,69 +2,166 @@
-second warehouse's name. If the warehouse can be resupplied by more than one warehouse, make sure
-to check those warehouses' boxes too. Finally, click :guilabel:`Save` to apply the setting. Now,
-Odoo knows which warehouses can resupply this warehouse.
-
-.. image:: resupply_warehouses/resupply-from-second-warehouse.png
+resupplied by the central warehouse. In the :guilabel:`Warehouse Configuration` tab, locate the
+:guilabel:`Resupply From` field, and check the box next to the central warehouse's name. If the
+warehouse can be resupplied by more than one warehouse, make sure to check those warehouses' boxes
+too. Finally, click :guilabel:`Save` to apply the setting. Now, Odoo knows which warehouses can
+resupply this warehouse.
+
+.. example::
+   The central warehouse that will supply the shops is called `Central warehouse`. The
+   :guilabel:`Resupply From` field is set to this warehouse on the shop's warehouse configuration
+   page.
+
+.. image:: resupply_warehouses/warehouse.png
    :align: center
    :alt: Supply one warehouse with another in the Warehouse Configuration tab.
 
 Set route on a product
-======================
+----------------------
 
-After configuring which warehouse(s) to resupply from, a new route is now available on all product
-forms. The new route appears as :guilabel:`Supply Product from [Warehouse Name]` under the
-:guilabel:`Inventory` tab on a product form. Use the :guilabel:`Supply Product from [Warehouse
-Name]` route with a reordering rule or the make to order (MTO) route to replenish stock by moving
-the product from one warehouse to another.
+Products must also be configured properly in order for them to be transferred between warehouses.
 
-.. image:: resupply_warehouses/product-resupply-route-settings.png
-   :align: center
-   :alt: Route setting which enables a product to resupplied from a second warehouse.
+Go to :menuselection:`Inventory app --> Products --> Products` and select the desired product. In
+the :guilabel:`Inventory` tab, the new route appears as :guilabel:`X: Supply Product from Y` in the
+:guilabel:`Routes` section.
 
-When a product's reordering rule is triggered and the product has the :guilabel:`Supply Product
-from [Warehouse Name]` route set, Odoo automatically creates two pickings. One picking is a
-*delivery order* from the second warehouse, which contains all the necessary products, and the
-second picking is a *receipt* with the same products for the main warehouse. The product move from
-the second warehouse to the main warehouse is fully tracked in Odoo.
+.. note::
+   *X* is the store's warehouse that receives products, and *Y* is the warehouse that sends
+   products.
 
-On the picking/transfer records created by Odoo, the :guilabel:`Source Document` is the product's
-reordering rule. The location between the delivery order and the receipt is a transit location.
+Tick the :guilabel:`X: Supply Product from Y` checkbox, which is intended to be used with the |MTO|
```

Comment:
oh yes, this is an [abbreviation](https://www.odoo.com/documentation/16.0/contributing/documentation/rst_guidelines.html#abbreviations) (:

It definitely looks like a link because of the dotted underline, but this is indeed expected behavior :sweat_smile: 

![image](https://github.com/user-attachments/assets/dcd7c82e-e0ed-4a08-8560-c6a779db2efb)

When we hover over the abbreviation, we get to see the fully spelled out version. At the top of the doc, I've defined that whenever I write `|MTO|`, it's replaced with the `abbr` code
|
![image](https://github.com/user-attachments/assets/262969c3-8d0e-45f8-8050-8ee5ad6fce41)